index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>

The word 'please' could have been added to make it more polite. So it could have been Game over, please refresh to play again 🧑‍💻 🤖!
```